code/datums/greyscale/json_configs/scarf.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
the fact scarves and largescarves didnt just share a .json is kind of upsetting :(